src/components/SupportIcon/SupportIcon.tsx

```diff
@@ -0,0 +1,79 @@
+import { LinearProgress, Box, Grid } from '@mui/material';
+import { withStyles } from '@mui/styles';
+import React from 'react';
+import Typography from '../Typography';
+
+export interface SupportIconProps {
+  variant: string; // supporting or disputing
+  value: number; // between 0 and 100
+  count: number;
+  label: string;
+}
+
+const BorderLinearProgress = withStyles({
+  root: {
+    height: 40,
+    width: 118,
+  },
+})(LinearProgress);
+
+const SupportIcon: React.FC<SupportIconProps> = ({
+  value,
+  variant,
+  label,
+  count,
+}) => {
+  const linearProgressColor = () => {
+    switch (variant) {
+      case 'supporting':
+        return 'success';
+      case 'disputing':
+        return 'error';
+      case 'neutral':
+        return 'warning';
+    }
+  };
+
+  return (
+    <Grid container direction="column" alignItems="center" spacing={1}>
+      <Grid item>
+        <Box position="relative" display="inline-flex">
```

Comment:
Does these not need a border ? My browsers (chrome, safari, firefox) do not render a border?

Reply:
Originally this component does not have a border. I think best to ask @tobiastykvart if I should add border, since I am just copying everything from project

Reply:
It doesn't need a border here I think since it's an icon. In Topics this is put inside a button which has a border, but we could use this icon in other contexts as well without a border.